betelgeuse.py

```diff
@@ -237,6 +237,7 @@ def add_test_case(args):
         # automated, manualonly, and notautomated
         auto_status = 'automated' if test.automated else 'notautomated'
         caseposneg = 'negative' if 'negative' in test.name else 'positive'
+        casecomponent = test.case_component if test.case_component else '-'
```

Comment:
The test object comes from Testimony parsing and this will generate an exception since it is not defined https://github.com/SatelliteQE/testimony/blob/master/testimony/__init__.py#L88-L106.

Testimony will not map to every Polarion field and as a counterpart it offers the `unexpected_tags` attribute which is a dict containing all not recognized fields. That said, in order to get the case component will be needed to use `test.unexpected_tags.get('casecomponent', '-')`. By doing that, if you have `@casecomponent` defined on your docstring it will get by Testimony and then by Betelgeuse.

With this new Testimony improvement we are going to be able to map all Polarion fields. The only thing that we should enforce is to name the same way pylarion expects in order to maintain the same meaning.